frontend/packages/dev-console/src/utils/pipeline-augment.ts

```diff
@@ -421,9 +424,24 @@ export const getRunStatusColor = (status: string): StatusMessage => {
 export const truncateName = (name: string, length: number): string =>
   name.length < length ? name : `${name.slice(0, length - 1)}...`;
 
-export const getTaskStatus = (pipelinerun: PipelineRun, pipeline: Pipeline): TaskStatus => {
-  const totalTasks =
-    pipeline && pipeline.spec && pipeline.spec.tasks ? pipeline.spec.tasks.length : 0;
+export const getPipelineFromPipelineRun = (pipelineRun: PipelineRun): Pipeline => {
+  if (!pipelineRun?.status?.pipelineSpec) {
```

Comment:
It's probably possible to pass in a inlineSpec for Pipeline, can we also check to see if the label for Pipeline is here on the PipelineRun?

```suggestion
  const pipelineName = pipelineRun.metadata.labels[TektonResourceLabel.pipeline];
  if (!pipelineName || !pipelineRun?.status?.pipelineSpec) {
```